opesci/staggeredgrid.py

```diff
@@ -915,12 +922,15 @@ def stress_loop(self):
                         + ccode(kernel.xreplace({self.t+1: self.time[1], self.t: self.time[0]})) + ';\n'
             dict1 = {'i': i, 'i0': i0, 'i1': i1, 'body': body}
             body = render(tmpl, dict1)
-            if self.ivdep and d == self.dimension-1:
+            if not self.pluto and self.ivdep and d == self.dimension-1:
                     body = '%s\n' % self.compiler._ivdep + body
-            if self.simd and d == self.dimension-1:
+            if not self.pluto and self.simd and d == self.dimension-1:
                     body = '#pragma simd\n' + body
 
-        if self.omp:
+        if self.pluto:
+            body = '#pragma scop\n'+body
+            body += '#pragma endscop'
+        elif self.omp:
```

Comment:
Ok, agreed let's start with exploring spatial tiling on a single loop. Still can you please move the `scop` pragmas to the template itself (guarded by an `%if pluto==True`). This way it is easier to identify the tiling regions for experimentation and move them if required.